docs/resim/metrics/batch_metrics.md

```diff
@@ -1,5 +1,57 @@
 # Batch metrics
 
-Our batch metrics system is currently functional but in an early phase where the interface is not fully finalized - as a result, the docs are a WIP. 
+Batch metrics are very similar to job metrics, but are computed after all the job metrics are computed, using the job-level metrics and metrics data as the "input."
 
-**If you are a customer and have any questions, feel free to reach out!**
+> Example Batch Metric 1: a weighted average of a job-level scalar metric, across all the jobs present.
+> Example Batch Metric 2: a histogram of a single series that is computed by all metrics jobs, after merging these series into one by appending.
```

Comment:
Formatting on this quote block seems a bit strange to me. Maybe add an extra line if you want each to show up on its own line? I've only looked at the GH markdown rendering though.